frontend/src/components/SearchBar.vue

```diff
@@ -70,8 +71,8 @@ export default defineComponent({
   setup () {
     const {push} = useRouter()
 
-    const tickerName = ref('')
-    const symbols: {data: AssetSymbol[]}    = reactive({data: []})
+    const tickerName                     = ref('')
```

Comment:
Doesnt this need to be a ref?

Reply:
Mmmm, yes, its the `v-model` of the input.